pallets/subtensor/src/lib.rs

```diff
@@ -780,7 +780,7 @@ pub mod pallet {
     pub type HotkeyEmissionTempo<T> =
         StorageValue<_, u64, ValueQuery, DefaultHotkeyEmissionTempo<T>>;
     #[pallet::storage]
-    /// Map ( hot ) --> emission | Accumulated hotkey emission.
+    /// Name corrected: PendingdHotkeyEmission => PendingHotkeyEmission
```

Comment:
Do we need this comment ?

Reply:
It's useful for people who index the chain like me. When I search the updated name there's nothing relevant, but the old name gives results:

- https://github.com/opentensor/subtensor/issues?q=is%3Aissue+PendingHotkeyEmission+
- https://github.com/opentensor/subtensor/issues?q=is%3Aissue+PendingdHotkeyEmission+

I happened to be aware of this typo, but if I wasn't and wanted to understand why my storage calls weren't working in earlier blocks this comment in the code would make it a lot easier to discover the reason.